src/styles/variables.js

```diff
@@ -150,4 +150,6 @@ export default {
     hoverDimValue: 1,
     pressDimValue: 0.8,
     qrShareHorizontalPadding: 32,
+
+    baseMenuItemHeight: 64,
```

Comment:
NAB
1. Why not use the existing `optionRowHeight` or `listItemHeightNormal`
2. If it's specific to a component why do we have a generic name `baseMenuItemHeight`?
3. Unnecessary line break